.gitignore

```diff
@@ -4,6 +4,7 @@
 example.tf
 terraform.tfplan
 terraform.tfstate
+.terraform.lock.hcl
```

Comment:
is this from a test that can be cleaned up? I'd actually like to remove files from gitignore that can alter behavior, as we've had many tests over the years that fail due to ignored files, which of course you can't track down using git.

Reply:
No, not a test, but excellent point; an accidental cli command (wrong directory) is what prompted adding this. I can remove it, I was just acting on the observation that we've added many other "standard" terraform artifacts in our .gitignore. 🤷

Reply:
(sometimes I get _obsessively_ consistent, which isn't always helpful!)